packages/valaxy/node/cli/new.ts

```diff
@@ -0,0 +1,39 @@
+import type { Argv } from 'yargs'
+
+export const newPost = (cli: Argv<{}>) => {
+  cli.command(
+    'new <title>',
+    'Draft a new post',
+    (args) => {
+      args.usage('$0 <title> -p [path] -l [layout]')
+        .positional('title', {
+          describe: 'The title of the new post',
+          required: true,
+        })
+        .option('path', {
+          alias: 'p',
+          type: 'string',
+          describe: 'the path to generate new post',
+          default: 'post',
+        })
+        .option('slug', {
```

Comment:
In fact, I think the name of `slug` a bit odd (although Hexo uses it). Do you have any other ideas?

The current URL is actually automatically generated based on the path.

Reply:
I thought so, if we are not using the mechanism we should remove this one instead.

Reply:
So let's remove it first, and then I'll merge the PR.

Reply:
Done.